views/student-analytics.pug

```diff
@@ -1,267 +1,281 @@
-#student-analytics-card
-  .card.white.padding-left-10.padding-right-10
+.container.hidden#student-analytics-card
+  if isAdmin
```

Comment:
why are we moving this here, were there any problems where it was previously?

Reply:
it was easier to make the default class that way